cmd/socat/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/matejvasek/socat
```

Comment:
Perhaps should be `github.com/knative-sandbox/kn-plugin-func/cmd/socat`

Reply:
I am not sure. Because I want it to work in Docker file as standalone CLI app, it must not be sub-module of the func project. Probably it should be even in separate repo.